scripts/install_avalanchego_release.sh

```diff
@@ -27,20 +27,66 @@ if [[ ${GOOS} == "darwin" ]]; then
   AVAGO_DOWNLOAD_PATH=${BASEDIR}/avalanchego-macos-${VERSION}.zip
 fi
 
-AVALANCHEGO_BUILD_PATH=${AVALANCHEGO_BUILD_PATH:-${BASEDIR}/avalanchego-${VERSION}}
+# TODO: Please read:
+# not sure why the version isn't included in CI currently... open to suggestions on how to handle this better
+AVALANCHEGO_BUILD_PATH=${AVALANCHEGO_BUILD_PATH-${VERSION}:-${BASEDIR}/avalanchego-${VERSION}}
```

Comment:
is the question why do we install this in a directory without the version as a suffix?

Reply:
Yeah... made me think that it's being used elsewhere.

Reply:
Currently we use this as a utility in local development to install a release of AvalancheGo, but we use it more frequently in CI to download a specific release that Subnet-EVM depends on to run its e2e tests.

In CI, we populate this environment variable here: https://github.com/ava-labs/subnet-evm/blob/master/.github/workflows/ci.yml#L77 and then pass the same variable in to run our ginkgo e2e tests here: https://github.com/ava-labs/subnet-evm/blob/master/.github/workflows/ci.yml#L83 (we could definitely clean this up a bit).

Just noticed we actually do something else that is weird here which is we use `BASE_DIR` instead of `BASEDIR` in one place in the script (on master) here: https://github.com/ava-labs/subnet-evm/blob/master/scripts/install_avalanchego_release.sh#L21.

Running this script locally now I'm running into the issue that instead of installing in `/tmp/` it's actually installing in the Subnet-EVM directory in the build path: `v1.9.11:-/tmp/avalanchego-release/avalanchego-v1.9.11`.

It looks like this is because the syntax `${var:-<alternative>}` checks if `var` is populated and uses it if so, but otherwise uses `<alternative>`. Since this change is not using a single variable, it seems to just use the whole string `v1.9.11:-/tmp/avalanchego-release/avalanchego-v1.9.11` instead of putting it in `/tmp`.

Reply:
Okay, take a look now. I left the commit as a fixup and will squash once I get the green light. Either we keep these changes or I can figure out a better way to deal with the branching logic.